wls-vorfaelleundvorkommnisse-service/src/test/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/rest/ereignis/EreignisControllerIntegrationTest.java

```diff
@@ -0,0 +1,166 @@
+
+            val mockedEreignisModelList = List.of(TestdataFactory.CreateEreignisModel.withEreignisart(EreignisartModel.VORFALL));
+            val mockedEreignisseWriteModel = TestdataFactory.CreateEreignisseWriteModel.withData(wahlbezirkID, mockedEreignisModelList);
+            val mockedEreignisList = TestdataFactory.CreateEreignisEntity.listFromModel(mockedEreignisseWriteModel);
+            val mockedEreignisseToLoad = ereignisModelMapper.toEreignisseModel(wahlbezirkID, false, true, mockedEreignisModelList);
+            ereignisRepository.saveAll(mockedEreignisList);
+
+            val request = MockMvcRequestBuilders.get("/businessActions/ereignisse/wahlbezirkID");
+            val response = api.perform(request).andExpect(status().isOk()).andReturn();
+            val responseBodyAsDTO = objectMapper.readValue(response.getResponse().getContentAsString(), WahlbezirkEreignisseDTO.class);
+
+            val expectedResponseDTO = ereignisDTOMapper.toDTO(mockedEreignisseToLoad);
+            Assertions.assertThat(responseBodyAsDTO).isEqualTo(expectedResponseDTO);
+        }
+    }
+
+    @Nested
+    class PostEreignisse {
+
+        @Test
+        @WithMockUser(authorities = { Authorities.SERVICE_POST_EREIGNISSE, Authorities.REPOSITORY_DELETE_EREIGNISSE, Authorities.REPOSITORY_WRITE_EREIGNISSE })
+        void should_save_list_of_ereignis_entities_when_new_data_successfully_saved() throws Exception {
+            val wahlbezirkID = "wahlbezirkID";
+
+            val mockedEreignisDtoList = List.of(
+                    TestdataFactory.CreateEreignisDto.withData(),
+                    TestdataFactory.CreateEreignisDto.withData(),
+                    TestdataFactory.CreateEreignisDto.withData());
+            val mockedEreignisseWriteDto = TestdataFactory.CreateEreignisseWriteDto.withData(mockedEreignisDtoList);
+
+            val request = createPostWithBody(wahlbezirkID, mockedEreignisseWriteDto);
+            val response = api.perform(request).andExpect(status().isOk()).andReturn();
+
+            SecurityUtils.runWith(Authorities.REPOSITORY_READ_EREIGNISSE);
+            val savedEreignisse = ereignisRepository.findByWahlbezirkID(wahlbezirkID);
+
+            val expectedSavedEreignisse = mockedEreignisseWriteDto.ereigniseintraege().stream()
+                    .map(ereignisDto -> new Ereignis(wahlbezirkID, ereignisDto.beschreibung(), ereignisDto.uhrzeit(),
+                            EreignisartEntity.fromDto(ereignisDto.ereignisart())))
+                    .toList();
+            Assertions.assertThat(response.getResponse().getContentAsString()).isEmpty();
+            Assertions.assertThat(savedEreignisse.size()).isEqualTo(expectedSavedEreignisse.size());
+            Assertions.assertThat(savedEreignisse).allSatisfy(ereignis -> {
+                Assertions.assertThat(ereignis.getClass().equals(Ereignis.class));
+                Assertions.assertThat(ereignis.getWahlbezirkID().equals(wahlbezirkID));
+            });
```

Comment:
- Der Klassencheck ist nicht notwendig
- Assert hatte keinen Check
```suggestion
            Assertions.assertThat(savedEreignisse).allSatisfy(ereignis -> {
                Assertions.assertThat(ereignis.getWahlbezirkID()).isEqualTo(wahlbezirkID);
            });
```